server/server.go

```diff
@@ -2209,6 +2211,10 @@ func (s *Server) generateDefaultDecisionPath() string {
 	return p
 }
 
+func (s *Server) updateConfig(config *cfg.Config, cacheConfig *iCache.Config) {
```

Comment:
Nit: We are not really using the OPA config here only the cache config. What do you think about changing this to something like `cacheConfigUpdated(cacheConfig *iCache.Config) `

Reply:
I'm fine with that, added it for the sake of future-proofing it, but maybe it's just YAGNI.